app/lib/actions/katello/content_view_version/import_library.rb

```diff
@@ -0,0 +1,71 @@
+# rubocop:disable Lint/SuppressedException
+module Actions
+  module Katello
+    module ContentViewVersion
+      class ImportLibrary < Actions::EntryAction
+        def plan(organization, path:, metadata:)
+          action_subject(organization)
+          unless SmartProxy.pulp_primary.pulp3_repository_type_support?(::Katello::Repository::YUM_TYPE)
+            fail ::Katello::HttpErrors::BadRequest, _("This API endpoint is only valid for Pulp 3 repositories.")
```

Comment:
I'd rather see this error raised in the controller or raise here but not speak about an API ie

`fail _("This action will become available after the Pulp 3 content migration")`